File: archive/fizz-buzz.js
var fizzBuzz = function (n) {
  let ans = []
  for (let i = 0; i < n; i++) {
    ans.push(`${i + 1}`)

    if (ans[i] % 3 === 0) {
      if (ans[i] % 5 === 0) {
        ans[i] = "FizzBuzz"
      } else {
        ans[i] = "Fizz"
      }
    } else if (ans[i] % 5 === 0) {
      ans[i] = "Buzz"
    }
  }

  return ans
};

console.log(fizzBuzz(3))

File: fizz-buzz.go
func fizzBuzz(n int) []string {
   ans := []string{}

   for i:=1; i <= n; i++ {
       if (i % 15 == 0) {
           ans = append(ans, "FizzBuzz")
       } else if (i % 5 == 0) {
           ans = append(ans, "Buzz")
       } else if (i % 3 == 0) {
           ans = append(ans, "Fizz")
       } else {
           ans = append(ans, strconv.Itoa(i))
       }
   }

   return ans
}